app/src/test/java/io/intrepid/skeleton/testutils/TestPresenterConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package io.intrepid.skeleton.testutils;
+
+import android.support.annotation.NonNull;
+
+import io.intrepid.skeleton.base.PresenterConfiguration;
+import io.intrepid.skeleton.rest.RestApi;
+import io.intrepid.skeleton.settings.UserSettings;
+import rx.schedulers.TestScheduler;
+
+public class TestPresenterConfiguration extends PresenterConfiguration {
+    public TestPresenterConfiguration(@NonNull TestScheduler ioScheduler,
+                                      @NonNull TestScheduler uiScheduler,
+                                      @NonNull UserSettings userSettings,
+                                      @NonNull RestApi restApi) {
+        super(ioScheduler, uiScheduler, userSettings, restApi);
+    }
+
+    public TestScheduler getTestIoScheduler() {
```

Comment:
You can't override by just changing the return type, hence the new method to return a type-safe TestScheduler.